.github/workflows/ci-windows.yml

```diff
@@ -7,7 +7,34 @@ permissions:
 
 jobs:
   Windows:
-    name: Windows
+    name: Windows (Native OPAM)
+    runs-on: windows-latest
+    steps:
+      - name: 🔭 Checkout code
+        uses: actions/checkout@v4
+
+      - name: 😈 Evil hack
+        run: rm rocq-prover.opam
+
+      - name: 🐫 Setup OCaml
+        uses: ocaml/setup-ocaml@v3
+        with:
+          ocaml-compiler: 4.14
+          dune-cache: true
+          opam-local-packages: rocq-*.opam
+
+      - name: 🐫🐪🐫 Get Rocq dependencies
+        run: opam install --deps-only ./rocq-*.opam
+
+      - name: 🧱 Build Rocq
+        run: opam exec -- make world
+
+      - name: 🐛 Test Rocq
+        run: opam exec -- make -C test-suite
```

Comment:
if we stick to -C we should also pass some -j
(IDK what the workers can handle but at least -j 2 I guess)

Reply:
`echo %NUMBER_OF_PROCESSORS%` should tell us what we got available on Win.

I havent followed the latest developments too closely, let's use the one you think is better.